### PR TITLE
Remove Google Analytics page counts

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR removes Google Analytics page view tracking from our app.

## Related Tickets & Documents

No ticket, this came out of a Slack comment by @benhalpern:

> I'd say at this point [Google Analytics for tabulating page views] is probably not necessary now that we've adopted our native way of tracking pageviews and we can probably drop this dependency altogether. It's kind of complicated and weird. @oss I'd say we can look to remove this whole part of our app as it's not entirely necessary.

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?

* Remove `GA_SERVICE_ACCOUNT_JSON` environment variable from Hereoku

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by citizen428** [comment]
@benhalpern After I optimistically got started with this, I realized I'm missing a lot of context, so maybe you can help me a bit by filling in the blanks? Am I removing the right things here? Will we still need `page_views_count` on articles? etc.

**Comment by benhalpern** [comment]
Yeah, we still need `page_views_count`, but we no longer need `GoogleAnalytics` entirely.

Right now we check Google Analytics and our own sum of page view count equivalence to tabulate the score (and take the bigger of the two for optimism 🤷‍♂ ), but this is largely because we didn't always have the in-house tabulation of page views so we relied on GoogleAnalytics to backdate.

GA also has better gaming preventions, we don't really have a lot of that (but perhaps we could apply #6841 to help?)

New communities won't have the backdate/backfill problems. We'll probably need some way of ensuring we never accidentally lose track of historical data, but we don't necessarily need the Google Analytics API to provide the core functionality, as our own `page_views` endpoint seems to work well enough (of course, this means we lose tabulation when we have backend downtime, but just means we need to be that much more reliable).

If the choice to remove it proves ill-fated, we could always introduce it back in, as the stored counts in GA will still be there when we go back, as we're not removing Google Analytics tracking altogether.

Getting this properly hooked up via Google Cloud is a burden I'm very glad to to pass on to other communities.

**Comment by citizen428** [comment]
@benhalpern Thanks for the context. I removed what I *think* can be removed, please sanity check.

File: Gemfile [hunk 1]
```diff
@@ -49,7 +49,6 @@ gem "fog-aws", "~> 3.6" # 'fog' gem to support Amazon Web Services
 gem "front_matter_parser", "~> 0.2" # Parse a front matter from syntactically correct strings or files
 gem "gemoji", "~> 4.0.0.rc2" # Character information and metadata for standard and custom emoji
 gem "gibbon", "~> 3.3" # API wrapper for MailChimp's API
-gem "google-api-client", "~> 0.37" # Client for accessing Google APIs
```
Comment (rhymes) [inline Gemfile:52]: 🔥

**Comment by citizen428** [comment]
@benhalpern I think this needs a final sanity check by you 😺

**Pull request merged**